tests/article.stat.test.js

```diff
@@ -0,0 +1,41 @@
+import chai from 'chai';
+import chaiHttp from 'chai-http';
+
+import server from '../src/index';
+import statusCodes from '../src/helpers/constants/status.codes';
+import signup from '../src/helpers/tests/signup';
+
+
+chai.use(chaiHttp);
+chai.should();
+
+const { OK } = statusCodes;
+const expectedStat = {
+  reads: 3,
+  shares: 2,
+  comments: 0
+};
+
+const token = signup();
+
+describe('Statistics', () => {
+  it('Should send article\'s stat', (done) => {
+    chai
+      .request(server)
+      .get('/api/articles/How-to-create-sequalize-seeds/stats')
+      .end((err, res) => {
+        res.body.article.stats.should.include(expectedStat);
+        done();
+      });
+  });
+  it('Should increase the number of reads', (done) => {
```

Comment:
@alainburindi but you're not checking that the number of reads has actually increased